scripts/scil_dwi_detect_volume_outliers.py

```diff
@@ -67,14 +63,13 @@ def main():
     dwi = nib.load(args.in_dwi)
     data = dwi.get_fdata()
 
-    b0_thr = check_b0_threshold(args.force_b0_threshold,
-                                bvals.min(), args.b0_thr)
+    args.b0_threshold = check_b0_threshold(bvals.min(), args)
```

Comment:
Why putting `args.b0_threshold = ` here?

Reply:
That's the way the --skip_b0 works. If you minimal bvalue is above the tolerance, it returns the minimal b-value instead.